src/lib/Scenes/Home/Components/TabBar.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react"
```

Comment:
I'm going to need this component in the settings page I think, so this should ideally go inside a shared space

Reply:
This tab bar component specifically follows the styling for the home v3 UI. However, if we already have the designs for the settings page, we can make sure it's generic enough to work for both views.

Reply:
@orta I don't think you will need it

Reply:
@orta oops nvm yes, yes you will lol